frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/configs/createOrganizationConfig.js

```diff
@@ -0,0 +1,318 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+export const createOrganizationConfig = () => {
+    const { t } = useTranslation()
+
+    const userInfo = Digit.UserService.getUser();
+    const tenantId = Digit.ULBService.getCurrentTenantId();
+    const ULB = Digit.Utils.pt.getCityLocale(tenantId);
```

Comment:
also change this Digit.Utils.pt.getCityLocale to  Digit.Utils.locale.getCityLocale

Reply:
done